pkg/plugins/golang/deploy-image/v1alpha1/scaffolds/internal/templates/config/samples/crd_sample.go

```diff
@@ -36,6 +37,7 @@ func (f *CRDSample) SetTemplateDefaults() error {
 		f.Path = filepath.Join("config", "samples", "%[group]_%[version]_%[kind].yaml")
 	}
 	f.Path = f.Resource.Replacer().Replace(f.Path)
+	fmt.Println(f.Path)
```

Comment:
Debugging?

Reply:
No. That is to ensure the default standard of printing the path when we do the scaffold as done in the other actions.